### ISSUE TITLE
`addEndDelay` keyword for midi export

### ISSUE THREAD

**Issue by jacobtylerwalls** [cuthbertLab/music21#1141]
**Motivation**
Just like `addStartDelay=False` can be sent to `.show('midi', addStartDelay=...)`, we could create `addEndDelay` to skip the trailing rest made by `getEndEvents()`.

**Feature summary**
See stack overflow [post](https://stackoverflow.com/a/69339207/14818839)

**Intent**

[ ] I plan on implementing this myself.
[ ] I am willing to pay to have this feature added.
[x] I am starting a discussion with the hope that community members will volunteer their time to create this. I understand that individuals work on features of interest to them and that this feature may never be implemented.

***

Great low-complexity PR for somebody interested in MIDI. Can largely follow the existing model of `addStartDelay`.

**Comment by mscuthbert** [comment]
Yes, very welcome feature.  End delay is on by default because a lot of MIDI players otherwise render a sharp pop-end to cutting off sound from a still-playing waveform.
